script/Ciel/Day 8/08_01_CIEL08_1D.txt

```diff
@@ -269,158 +290,190 @@ I want to know more about her.
+the thing that I had been gazing down on mere moments
 }
 [sha:38c9bc47d48ed0729afb2b67275fa5a699479c51]{
 -- Page 123, Offset 18574.
 -- その実、見上げるほど巨大だった。
-Was actually a giant so large I had to gaze up to it.
+ago is, in reality, so large that I now have to turn my gaze upward instead.
 }
 [sha:b4082df3f5a901c8299c870a8a76c2f5b5fcfa3c]{
 -- Page 124, Offset 18575.
 -- 死体が膨張していく。認知が津波になる。
-Her corpse is expanding. The weight of my actions comes crashing down on me like a tsunami.
+Her corpse looms larger and larger.
+ The weight of my actions comes crashing down on me like a tsunami.
 }
 [sha:c9b3efbb1759f1add6a75574904fdffd3fcc8579]{
 -- Page 124, Offset 18576.
 -- 何もかも手遅れだった。手に負えないほど圧倒的だった。
-It's too late to do anything. It's so overwhelming that I feel helpless.
+It's too late for me to do anything.
+ It's so overwhelming that I feel utterly hopeless.
 }
 [sha:2e5cd497d463e2e9df2699e76df2ae0ebdd010af]{
 -- Page 124, Offset 18577.
 -- 何の比喩でもなく、溺れるような情報量だった。
-It is no metaphor, I could have drowned under that much information.
+In a very literal sense, I could drown under all this information.
 }
 [sha:f075e48e34db81e9f7bb5b88fe8e076e59529625]{
 -- Page 124, Offset 18578.
 -- 逃げる事も、息をする事もできなかった。
-I couldn't even escape, couldn't even breathe.
+I can't escape, I can't even breathe.
 }
 [sha:259f64764d024ee61c98266001a15733404cf16a]{
 -- Page 124, Offset 18579.
 -- あまりにも膨大な、今まで機能として／道理として学んでおきながら目を瞑っていた、それは命の仕組みだった。
-// Confusing sentence, I'm not sure I got it right
-It was the colossal workings of life, the faculties, the natural way of things that I had up to now studied with my eyes closed.
+// G: Confusing sentence, I'm not sure I got it right
+// R: ditto. My take rendered here as well, but no clue which is totally correct (if either). Petrikow pls.
+// It was the colossal workings of life, the faculties, the natural way of things that I had up to now studied with my eyes closed.
+This is something I'd learned of in principle, but had pretended not to see――the monumental workings of life itself.
 }
 [sha:442c7ca064762f9a8d154d9a4eb5e7d34e1ae210]{
 -- Page 125, Offset 18580.
 -- 意識が<矮|ちい>さくなる。自我が<些|わずか>になる。
-My consciousness is withering. My sense of self is being trivialized.
+My consciousness grows smaller and smaller.
+ My sense of self becomes less and less significant.
 }
 [sha:7632bba247b4373abbb0369f0ea3cd6efe044a57]{
 -- Page 125, Offset 18581. Mods: @n.
 -- 俺という個が、自らの低級さに耐えられず<萎|しぼ>んでいく。
-The object that I am, unable to bear its own vulgarity, is shrinking.
+Unable to bear my own vulgarity, I begin to waste away.
 }
 [sha:f60cf322ba68fa64ddd374cf150b39de3af4a6ff]{
 -- Page 125, Offset 18582.
 -- それを否定する材料はなく、<己|つみ>を<暈|ぼか>せば暈すほど、世界の拡大は止まらなかった。
-Without basis for denial, the more I try to blur <my sins|m y s e l f>, the more the world around me expands.
+Without any basis on which to deny them, the more I try to blot out my <sins|s e l f  >,
+ the larger the world balloons around me.
```

Comment:
The way this sentence is constructed in Japanese is almost ungrammatical. It's kinda absurd.
There's something more I'd like to do with it, but its a very difficult sentence. That said, the 'balloons around me' expression feels inappropriate here.